run.py

```diff
@@ -83,8 +85,14 @@ def run_one_step(func, nwarmup=WARMUP_ROUNDS):
         t0 = time.time_ns()
         func()
         t1 = time.time_ns()
+        wall_latency = t1 - t0
         print('{:<20} {:>20}'.format("CPU Total Wall Time:", "%.3f milliseconds" % ((t1 - t0) / 1_000_000)), sep='')
 
+    # if model_flops is not None, output the TFLOPs per sec
+    if model_flops:
+        tflops = wall_latency / 1_000_000_000 * model_flops / 1.0e12
```

Comment:
a bit weird here, the way we count tflops is batch_iter * flops_count/ time_sec/1.0e12

Reply:
Yeah, notice it's TFLOPs per second so we need to divide by latency?

Reply:
Fixed the formula. We always run a single batch in torchbench so `batch_iter` should always be 1.